host_cleanup.sh

```diff
@@ -21,3 +21,5 @@ ANSIBLE_FORCE_COLOR=true ansible-playbook \
 sudo rm -rf /etc/NetworkManager/dnsmasq.d/openshift.conf /etc/NetworkManager/conf.d/dnsmasq.conf
 sudo virsh net-destroy baremetal
 sudo virsh net-undefine baremetal
+sudo virsh net-destroy overcloud
+sudo virsh net-undefine overcloud
```

Comment:
Wouldn't these only be needed on a machine with a prior deployment via these scripts? tl;dr we could likely undo this in like a week?

Reply:
> Wouldn't these only be needed on a machine with a prior deployment via these scripts? tl;dr we could likely undo this in like a week?
Yes, if the respective virsh network doesn't exist - error will be printed and the script will continue executing.

Reply:
This is actually a bug in the quickstart roles I think, since the ./tripleo-quickstart-config/roles/environment/setup/tasks/main.yml file contains the task which creates this network, but it's not cleaned up in the destroy tasks.

Removing it here should be fine though, even though it's probably not strictly necessary unless the configuration of that network changes.